crates/ruff_python_formatter/src/type_param/type_param_type_var.rs

```diff
@@ -12,11 +12,15 @@ impl FormatNodeRule<TypeParamTypeVar> for FormatTypeParamTypeVar {
             range: _,
             name,
             bound,
+            default_value,
         } = item;
         name.format().fmt(f)?;
         if let Some(bound) = bound {
             write!(f, [token(":"), space(), bound.format()])?;
         }
+        if let Some(default_value) = default_value {
+            write!(f, [token("="), space(), default_value.format()])?;
+        }
```

Comment:
Similar to other suggestion.

```suggestion
        if let Some(default_value) = default_value {
            write!(f, [space(), token("="), space(), default_value.format()])?;
        }
```